TrainingOOP-PHP001/Point.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace TrainingOOPPHP001;
+
+class Point{
```

Comment:
Chú ý có space trước `{`
```suggestion
class Point {
```